src/ui/main.qml

```diff
@@ -441,6 +441,13 @@ Window {
         id: serverUnavailablePopup
     }
 
+    Connections {
+        target: VPNController
+        function onReadyToServerUnavailable() {
+            serverUnavailablePopup.open();
+        }
+    }
```

Comment:
I think this part was lost during a rebase with master. Also happy to create a separate PR to add this line back in.